src/include/distributed/connection_management.h

```diff
@@ -42,6 +42,9 @@
 /* application name used for connections made by run_command_on_* */
 #define CITUS_RUN_COMMAND_APPLICATION_NAME_PREFIX "citus_run_command gpid="
 
+/* application name prefix for move/split replication connections (no GPID) */
+#define CITUS_SHARD_TRANSFER_APPLICATION_NAME_PREFIX "citus_shard_"
```

Comment:
in IsCitusShardTransferBackend

Reply:
yes, but I cannot see this application name is set anywhere? I checked existing code in case we use it as hard coded manner, but still couldn't see. If we use in other places, should we use this macro there as well?